.github/workflows/config/spelling_allowlist.txt

```diff
@@ -176,6 +176,7 @@ discoverable
 discretization
 discretize
 discretized
+dtype
```

Comment:
dtype isn't a word. Wherever it is being used should be backticked (similar to the ndarray comment below).